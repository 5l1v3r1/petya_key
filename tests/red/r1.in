e4Q8ffSQY1yvicQi2FERwAoYQLfRPJ9b9pukbkSQ3Pk21KJr9DXZbfqXZDFKnnKZRPJ5MXvNAC9eajUsqRCNu43hQ2
