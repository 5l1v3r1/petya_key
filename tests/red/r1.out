rU95BhkeU4Eua2q3
